wear/src/main/res/layout/activity_contact_events.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context="com.alexstyl.specialdates.ContactEventsActivity"
+  tools:deviceIds="wear_round">
+
+  <LinearLayout
+    android:id="@+id/event_container"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_centerInParent="true"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <TextView
+      android:id="@+id/date_text"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:drawablePadding="8dp"
+      android:drawableTop="@drawable/ic_complication_contact_event"
+      android:fontFamily="sans-serif"
+      android:gravity="center"
+      android:textStyle="italic" />
```

Comment:
All the styling of the views should go to a `style` resource in this feature's `resource.xml` file.

Have a look at [upcoming-resources.xml](https://github.com/alexstyl/Memento-Namedays/blob/master/mobile/src/main/res/values/upcoming-resources.xml) to get started :)